.github/workflows/ci.yaml

```diff
@@ -46,6 +46,23 @@ jobs:
       - name: Integration Tests
         run: bazel --batch test //test/integration/...
 
+      - name: Gradle Build Tests on Generated Libraries
```

Comment:
Can we just split it into two steps? (1) Gradle Build Generated Storage Client Library; (2) Generate and Gradle Build Compute Client Library?
Otherwise, it implies like all client libraries are being tested.